string action
trajectory_msgs/JointTrajectoryPoint joint_goal
geometry_msgs/Pose cartesian_goal
float32 velocity
float32 gripper_speed
int32 gripper_width
bool gripper_lock
bool gripper_grip_box
---
bool success
---
float32 ping